sdk/eventgrid/azure-messaging-eventgrid/README.md

```diff
@@ -240,8 +240,8 @@ cloudEventClient.sendEvents(events);
 <!-- embedme ./src/samples/java/com/azure/messaging/eventgrid/samples/ReadmeSamples.java#L141-L154 -->
 ```java
 // Make sure that the event grid topic or domain you're sending to accepts the custom event schema.
-List<Object> events = new ArrayList<>();
-events.add(new HashMap<String, String>() {
+List<BinaryData> events = new ArrayList<>();
+events.add(BinaryData.fromObject(new HashMap<String, String>() {
```

Comment:
Do we have samples that use a simple POJO instead of HashMap?

Reply:
Will do this sample separately.